Documentation/installation/requirements-rke.rst

```diff
@@ -1,37 +1,7 @@
-To install Cilium on `Rancher Kubernetes Engine (RKE) <https://rancher.com/docs/rke/latest/en/>`_,
-perform the following steps:
+To install Cilium top of a **standalone** `Rancher Kubernetes Engine 1 (RKE1) <https://rke.docs.rancher.com/>`_
+or `Rancher Kubernetes Engine 2 (RKE2) <https://docs.rke2.io/>`_ cluster,
+follow the installation instructions provided in the dedicated
+:ref:`Installation using Rancher Kubernetes Engine <rke_install>` guide.
 
-.. note::
-
-   If you are using RKE2, Cilium has been directly integrated. Please see
-   `Using Cilium <https://docs.rke2.io/install/network_options#install-a-cni-plugin>`_
-   in the RKE2 documentation. You can use either method.
-
-**Default Configuration:**
-
-=============== =============== ==============
-Datapath        IPAM            Datastore
-=============== =============== ==============
-Encapsulation   Cluster Pool    Kubernetes CRD
-=============== =============== ==============
-
-**Requirements:**
-
-* Follow the `RKE Installation Guide <https://rancher.com/docs/rke/latest/en/installation/>`_
-  with the below change:
-
-  From:
-
-  .. code-block:: yaml
-
-     network:
-       options:
-         flannel_backend_type: "vxlan"
-       plugin: "canal"
-
-  To:
-
-  .. code-block:: yaml
-
-     network:
-       plugin: none
+If your RKE1/2 cluster is managed by Rancher (**non-standalone**), follow the
+:ref:`Installation using Rancher <rancher_managed_rke_clusters>` guide instead.
```

Comment:
The linked rancher_managed_rke_clusters page says that it's "solely meant for Rancher-managed RKE2 clusters", but here you also mention RKE1 clusters. Does Rancher support Cilium in RKE1 clusters?

Reply:
Good point. I have also tested RKE1; luckily, it's even easier -- added a separate "Note box".

I addressed it in https://github.com/cilium/cilium/pull/30178/commits/9f160c9ad1085564abd39109559ff772a3fd006e. I'll squash the two commits after your approval.